lsp-headerline.el

```diff
@@ -143,15 +143,18 @@ caching purposes.")
 (defun lsp-headerline--fix-image-background (image)
   "Fix IMAGE background if it is a file otherwise return as an icon."
   (if image
-      (let ((display-image (get-text-property 0 'display image)))
-        (if (listp display-image)
+      (let* ((display-image (get-text-property 0 'display image)))
```

Comment:
Is there a reason to use `let*` instead of `let`?

Reply:
No, added and then removed a var, I'll rollback this, thanks!